src/django/api/serializers.py

```diff
@@ -55,6 +55,16 @@ def _get_parent_company(claim):
         'name': claim.parent_company.name,
     }
 
+def is_embed_mode_active(serializer):
+    request = serializer.context.get('request') \
+        if serializer.context is not None else []
+
+    if request is not None or request.query_params is not None:
+        if request.query_params.get('embed') == '1':
+            return True
+
+    return False
```

Comment:
This function is based off of [this code](https://github.com/open-apparel-registry/open-apparel-registry/pull/1342/files#diff-9d16a5dca79134fade5a462b1028f0d3187305adcd0cc171e952baa8ebfce98eR705-R713). I'm not entirely sure of all the scenarios it is trying to protect against, but I thought the same conditions should be applied here.